src/univariate/continuous/normal.jl

```diff
@@ -75,6 +75,17 @@ kurtosis(d::Normal{T}) where {T<:Real} = zero(T)
 
 entropy(d::Normal) = (log2π + 1)/2 + log(d.σ)
 
+"""
+    kldivergence(p::Normal, q::Normal)
+
+See [KL Gaussian](https://en.wikipedia.org/wiki/Normal_distribution#Other_properties)
+"""
+function kldivergence(p::Normal, q::Normal)
+    μp, σp = params(p)
+    μq, σq = params(q)
+    return 0.5 * (abs2(σp / σq) + abs2((μp - μq) / σq) - 1 + 2 * (log(σq) - log(σp)))
+end
```

Comment:
Eliminating some computations:
```suggestion
    μp = mean(p)
    σ2p = var(p)
    μq = mean(q)
    σ2q = var(q)
    return ((μp - μq)^2 + σ2p) / σ2q - 1 + log(σ2q / σ2p)) / 2
```

Reply:
How about doing similarly to your other propositions and having a `sigma2p_over_sigma2q` ?

Reply:
There's no efficiency gain since you still have to define the squared difference of the means by sigma2q separately.